OWML.ModHelper/Logging/OutputFactory.cs

```diff
@@ -1,23 +1,18 @@
 ﻿using OWML.Common;
 
 
-namespace OWML.ModHelper
+namespace OWML.ModHelper.Logging
 {
     public class OutputFactory
     {
         public static IModConsole CreateOutput(IOwmlConfig owmlConfig, IModLogger logger, IModManifest manifest)
         {
-            if (CommandLineArguments.HasArgument(Constants.ConsolePortArgument))
-            {
-                return new ModSocketOutput(owmlConfig, logger, manifest);
-            }
-
             if (logger == null)
```

Comment:
Your changes in this file make the first OWML console logs (before unity) no longer be sent to the socket, because that writer is created with logger = null. It should only return `OutputWriter` if the consolePort param isn't present.

Reply:
this *should* be okay now?